cantonese/siuming.md

```diff
@@ -0,0 +1,5 @@
+小明係一個有為青年，不過就比較大頭蝦，成日唔記得事情。有一日，佢要搭飛機去美國讀書，同家人朋友道別後佢獨自走入禁區。當佢正煩惱緊去到美國應該食薄餅先定係漢堡包先既時候，佢發現佢漏左個好緊要既袋係離境大堂。呢個時候，如果你係小明既話，你會...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.